PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+(13:42) that doesn't have a cost ultimately ends up being political in nature so people closer to the money so called canton effect are going to be advantaged people with the purse strings to create more money have enormous political power lynn alden also takes a similar stance to adam back saying why she thinks proof of stake mimics stock-like assets more than monetary assets right so for if we're talking about what makes a good money it's something that is inherently simple an inherently small no decentralized
+
+(14:13) proof of work basically the the leanest code you can get and distribute as among as many machines as you can get and as immutable as you can get whereas as we layer things on top of it like proof of stake adds a lot of code complexity it also makes it use my prior example more like volatile memory than non-volatile memory needs to be continuously live in order to function as intended and so when you layer those things on top of it i would say that takes away a monetary premium and you you can argue that it adds equity value right so for example
+
+(14:44) when we look at apple stock right so we look at apple stock over the long term you know like this this this kind of meme of ultrasound money for ethereum you have this thing where it's mostly focused on whether it's inflating or deflating and what percentage right so for example apple stock is actually one of the most deflationary assets around it's it's got a 4.
+
+(15:05) 5 average annual deflation rate over the past 10 years they decreased their share account from 26 million to 16 million and that's of course one of the factors for why they did well on a per share basis but at the end of the day what it comes down to for because they're in equity it's the attractiveness of their ecosystem for customers right so it's not about how many shares they have because you have other ones like ibm that are also rapidly reducing their share count but because their ecosystems
+
+(15:30) unhealthy they're not really crewing that value to shares overall and so i think when it comes down to kind of what is trying to be money the thing that does almost nothing other than just be money is the best money and something that tries to do multiple things including proof of stake as one of the components inherently gives it less monetary premium in a proof of work system power is distributed between miners developers and individual nodes your ability to be a miner is based on your ability to put forth money and find low-cost
```

Comment:
Distinguere le frasi non del narratore `right so for if we're talking about what makes a good money it's something that is inherently simple [...] including proof of stake as one of the components inherently gives it less monetary premium`